_opam

```diff
@@ -0,0 +1,5 @@
+depexts: [
```

Comment:
That way, you simply do `oasis2opam <tarball URL>` and the opam metadata is automatically generated from your `_oasis` file (which has no provision for `depexts`).  This ensures that the OPAM metadata is kept in sync with the file that drives your build.